addons/mass_mailing/models/ir_model.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class IrModel(models.Model):
+    _inherit = 'ir.model'
+
+    is_mailing = fields.Boolean(
```

Comment:
"is_mailing"  could be a bit confusing.

Why not using the same syntax than the class property and make it "is_mailing_enabled" or something even more precise like "is_mass_mailing_compatible" / "has_mailing_capabilities" (last one is annoying to type though).

Reply:
Went for ``is_mailing_enabled``